spec/spec_helper.rb

```diff
@@ -9,4 +9,12 @@
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
+
+  config.disable_monkey_patching!
```

Comment:
Haha yeah not sure why they named it that. Basically just forces you to use `Rspec.describe` vs `describe` on the top level spec definition. Aka prevents exposing Rspec DSL globally